### PR TITLE
Rendering, Options, and Packet

### PR DESCRIPTION
**Pull request by CoolMineman**

### CODE REVIEW

File: mappings/net/minecraft/client/gui/hud/InGameHud.mapping [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+	FIELD field_10032 ticks I
+	FIELD field_10033 overlayMessage Ljava/lang/String;
+	FIELD field_10034 overlayRemaining I
+	FIELD field_10035 overlayTinted Z
+	FIELD field_10036 heldItemTooltipFade I
+	FIELD field_10037 heldItem Lnet/minecraft/class_2056;
+	METHOD method_9190 tick ()V
+	METHOD method_9193 render (FZII)V
+		ARG 1 tickDelta
+		ARG 2 inScreen
```
Comment (hYdos) [inline mappings/net/minecraft/client/gui/hud/InGameHud.mapping:19]: inScreen?

File: mappings/net/minecraft/client/render/BufferBuilder.mapping [hunk 2]
```diff
@@ -6,3 +6,20 @@ CLASS net/minecraft/class_1002 net/minecraft/client/render/BufferBuilder
...
 		ARG 5 z
+	METHOD method_9550 draw ()I
+	METHOD method_9552 uv (DD)V
+		ARG 1 u
+		ARG 3 v
+	METHOD method_9553 vertex (DDD)V
+		ARG 1 x
+		ARG 3 y
+		ARG 5 z
+	METHOD method_9554 vertexUv (DDDDD)V
```
Comment (hYdos) [inline mappings/net/minecraft/client/render/BufferBuilder.mapping:17]: ``vertexUV``

**Review by hYdos — CHANGES_REQUESTED** [review]

File: mappings/net/minecraft/client/render/BufferBuilder.mapping (hunk 2 above)
Comment (BoogieMonster1O1) [reply mappings/net/minecraft/client/render/BufferBuilder.mapping:17]: upstream calls it `uv`
see `BufferBuilder`'s `light(int)` and `overlay(int)` methods

Comment (BoogieMonster1O1) [reply mappings/net/minecraft/client/render/BufferBuilder.mapping:17]: upstream calls it `uv`
see `BufferBuilder`'s `light(int)` and `overlay(int)` methods

**Review by hYdos — APPROVED** [review]

**Pull request merged**